 <!-- vim: set ft=xml noet : -->
<launch>
 <arg name="UAV" default="crazyflie" />
 <arg name="UAV_Odom_Topic" default = "/crazyflie/odom_mpc" />


 	<node name = "Reference_Node" pkg = "flyboi_ethz" type = "Circle_Path_Perfect_Timer.py" output="screen">
		<param name="SubscribeTopic" value="$(arg UAV_Odom_Topic)" />
	</node>
	

</launch>
